akka-persistence-query/src/main/scala/akka/persistence/query/journal/leveldb/EventsByPersistenceIdPublisher.scala

```diff
@@ -181,7 +181,7 @@ private[akka] class CurrentEventsByPersistenceIdPublisher(
     deliverBuf()
     if (highestSeqNr < toSequenceNr)
       toSeqNr = highestSeqNr
-    if (highestSeqNr == 0L || (buf.isEmpty && currSeqNo > toSequenceNr))
+    if (highestSeqNr == 0L || (buf.isEmpty && currSeqNo > toSequenceNr) || currSeqNo == fromSequenceNr)
```

Comment:
so that extra condition means that nothing was replayed?
do we need the `highestSeqNr == 0L` ?